README.md

```diff
@@ -2,3 +2,8 @@ Sandbox for Game Engine experiments.
 
 ![](Docs/scene.png)
 ![](Docs/physics.gif)
+
+> Give someone a game engine and they deliver a game.  
+> Teach someone to make a game engine and they never deliver anything.  
```

Comment:
'... except several half-made game engines' maybe?
According to https://mobile.twitter.com/mikegeig/status/403517625545670656